.azuredevops/modulePipelines/ms.appconfiguration.configurationstores.yml

```
@@ -0,0 +1,55 @@
+name: 'AppConfiguration: ConfigurationStores'
```

```suggestion
name: 'App Configuration - ConfigurationStores'
```
Otherwise not supported

---

True for the` :` part. However for consistency I'd suggest without a space:
```suggestion
name: 'AppConfiguration - ConfigurationStores'
```

---

Updated to `name: 'AppConfiguration - ConfigurationStores'`